### DISCUSSION TITLE
Remove/edit entry from `$env.PATH`

### DISCUSSION THREAD

**Discussion by roaldarbol** [nushell/nushell#12739] [Q&A] [answered]
I've accidentally written something wrong into `$env.PATH`. Now I can't figure out how to remove it again, and I can't find the entries in any file (e.g. `$nu.env-path`). There's nothing about removing entries [in the documentation](https://www.nushell.sh/book/configuration.html#path-configuration) as far as I can tell.

Can anyone help here?

**Comment by fdncred** [comment, marked as answer]
Environment variables are inherited from the shell that nushell is launched from. If nushell is your login shell, then you specify env vars in the `$nu.env-path` like you have mentioned. Although, I believe any file you `source` or `use` can export environment variables so you may have to hunt for it.

**Reply by roaldarbol** [reply]
Ah, the problem was that I simply called `$env.PATH = ($env.PATH | split row (char esep) | append '/some/path')` from the shell, I didn't add it to `$nu.env-path` - so it actually didn't stay, and accordingly, there was nothing to remove. When I added it to `$nu.env-path`, I could of course simply remove it again.
